### PR TITLE
#97 Corrected some mistakes in toolchain for 3rd party.

### PR DESCRIPTION
**Pull request by gautierbureau**

### CODE REVIEW

File: dynawo/3rdParty/toolchain.sh [hunk 1]
```diff
@@ -221,7 +217,7 @@ while (($#)); do
 			LIBARCHIVE_HOME=`echo $1 | sed -e 's/--libarchive-install-dir=//g'`
       ;;
     --gtest-install-dir=*)
-      GTEST_INSTALL_DIR=`echo $1 | sed -e 's/--gtest-install-dir=//g'`
```
Comment (rosiereflo) [inline dynawo/3rdParty/toolchain.sh]: WHy is this modified here and not in libzip and libxml?

File: dynawo/3rdParty/toolchain.sh [hunk 2]
```diff
@@ -54,17 +54,13 @@ export_var_env_default() {
 }
 
 get_absolute_path() {
-  absolute=$(readlink -m $1)
-  echo "$absolute"
+  python -c "import os; print(os.path.realpath('$1'))"
```
[the remark this answers is not in the document]
Comment (gautierbureau) [reply dynawo/3rdParty/toolchain.sh]: I think we can assume it on Unix system, whereas readlink is only Linux. And we use Python a lot elsewhere !!

File: util/envDynawo.sh [hunk 3]
```diff
@@ -85,9 +85,14 @@ where [option] can be:
...
-HERE=$PWD
-SCRIPT=$(readlink -f $0)
-TOTAL_CPU=$(grep -c \^processor /proc/cpuinfo)
+if [ "`uname`" = "Linux" ]; then
+  TOTAL_CPU=$(grep -c \^processor /proc/cpuinfo)
+elif [ "`uname`" = "Darwin" ]; then
+  TOTAL_CPU=$(sysctl hw | grep ncpu | awk '{print $(NF)}')
+else
+  echo "OS not supported."
+  exit 1
```
[the remark this answers is not in the document]
Comment (gautierbureau) [reply util/envDynawo.sh]: In fact we could but in reality I am not sure someone on other platform would be avle to go through the script...

File: dynawo/3rdParty/toolchain.sh (hunk 1 above)
Comment (gautierbureau) [reply dynawo/3rdParty/toolchain.sh]: It is actually not used the same way
